app/controllers/jobseekers/profiles/qualified_teaching_status_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Jobseekers::Profiles::QualifiedTeachingStatusController < Jobseekers::ProfilesController
+  def edit
+    @profile = JobseekerProfile.find_by(jobseeker_id: current_jobseeker.id)
```

Comment:
Perhaps we should set @profile in the profiles controller and just use it in this controller.

In the profiles controller you'd have something like:
```

  private

  def profile
    @profile ||= JobseekerProfile.find_by(jobseeker_id: current_jobseeker.id)
  end
```

Then you can just use `profile` in the view and in the controller.